Add excerpts for currently defined words (#262)
selfdefined/web-app: 11ty/definitions/dumb.md

  text: 'ableist language'
  level: avoid
defined: true

> excerpt: non-verbal, non-speaking, silent, refraining from speaking or unable to speak; sometimes __mute__ which can also be ableist; colloquially used to mean unintelligent or senseless

Reviewer: ```suggestion
excerpt: avoid. ableist. colloquially used to mean unintelligent or senseless
```

Reviewer 2: We add avoid in the template already, I think by adding it to the excerpt it will show up twice.

> excerpt: non-verbal, non-speaking, silent, refraining from speaking or unable to speak; sometimes __mute__ which can also be ableist; colloquially used to mean unintelligent or senseless

Author: ```suggestion
excerpt: ableist term. avoid. colloquially used to mean unintelligent or senseless
```

Swapping the order to maintain consistency with other similar edits.
